setpci.c

```diff
@@ -271,60 +281,60 @@ static const struct reg_name pci_reg_names[] = {
   {       0, 0x0d, 1, "LATENCY_TIMER" },
   {       0, 0x0e, 1, "HEADER_TYPE" },
   {       0, 0x0f, 1, "BIST" },
-  {       0, 0x10, 4, "BASE_ADDRESS_0" },
-  {       0, 0x14, 4, "BASE_ADDRESS_1" },
-  {       0, 0x18, 4, "BASE_ADDRESS_2" },
-  {       0, 0x1c, 4, "BASE_ADDRESS_3" },
-  {       0, 0x20, 4, "BASE_ADDRESS_4" },
-  {       0, 0x24, 4, "BASE_ADDRESS_5" },
-  {       0, 0x28, 4, "CARDBUS_CIS" },
-  {       0, 0x2c, 2, "SUBSYSTEM_VENDOR_ID" },
-  {       0, 0x2e, 2, "SUBSYSTEM_ID" },
-  {       0, 0x30, 4, "ROM_ADDRESS" },
-  {       0, 0x34, 1, "CAPABILITIES" },
-  {       0, 0x3c, 1, "INTERRUPT_LINE" },
-  {       0, 0x3d, 1, "INTERRUPT_PIN" },
-  {       0, 0x3e, 1, "MIN_GNT" },
-  {       0, 0x3f, 1, "MAX_LAT" },
-  {       0, 0x18, 1, "PRIMARY_BUS" },
-  {       0, 0x19, 1, "SECONDARY_BUS" },
-  {       0, 0x1a, 1, "SUBORDINATE_BUS" },
-  {       0, 0x1b, 1, "SEC_LATENCY_TIMER" },
-  {       0, 0x1c, 1, "IO_BASE" },
-  {       0, 0x1d, 1, "IO_LIMIT" },
-  {       0, 0x1e, 2, "SEC_STATUS" },
-  {       0, 0x20, 2, "MEMORY_BASE" },
-  {       0, 0x22, 2, "MEMORY_LIMIT" },
-  {       0, 0x24, 2, "PREF_MEMORY_BASE" },
-  {       0, 0x26, 2, "PREF_MEMORY_LIMIT" },
-  {       0, 0x28, 4, "PREF_BASE_UPPER32" },
-  {       0, 0x2c, 4, "PREF_LIMIT_UPPER32" },
-  {       0, 0x30, 2, "IO_BASE_UPPER16" },
-  {       0, 0x32, 2, "IO_LIMIT_UPPER16" },
-  {       0, 0x38, 4, "BRIDGE_ROM_ADDRESS" },
-  {       0, 0x3e, 2, "BRIDGE_CONTROL" },
-  {       0, 0x10, 4, "CB_CARDBUS_BASE" },
-  {       0, 0x14, 2, "CB_CAPABILITIES" },
-  {       0, 0x16, 2, "CB_SEC_STATUS" },
-  {       0, 0x18, 1, "CB_BUS_NUMBER" },
-  {       0, 0x19, 1, "CB_CARDBUS_NUMBER" },
-  {       0, 0x1a, 1, "CB_SUBORDINATE_BUS" },
-  {       0, 0x1b, 1, "CB_CARDBUS_LATENCY" },
-  {       0, 0x1c, 4, "CB_MEMORY_BASE_0" },
-  {       0, 0x20, 4, "CB_MEMORY_LIMIT_0" },
-  {       0, 0x24, 4, "CB_MEMORY_BASE_1" },
-  {       0, 0x28, 4, "CB_MEMORY_LIMIT_1" },
-  {       0, 0x2c, 2, "CB_IO_BASE_0" },
-  {       0, 0x2e, 2, "CB_IO_BASE_0_HI" },
-  {       0, 0x30, 2, "CB_IO_LIMIT_0" },
-  {       0, 0x32, 2, "CB_IO_LIMIT_0_HI" },
-  {       0, 0x34, 2, "CB_IO_BASE_1" },
-  {       0, 0x36, 2, "CB_IO_BASE_1_HI" },
-  {       0, 0x38, 2, "CB_IO_LIMIT_1" },
-  {       0, 0x3a, 2, "CB_IO_LIMIT_1_HI" },
-  {       0, 0x40, 2, "CB_SUBSYSTEM_VENDOR_ID" },
-  {       0, 0x42, 2, "CB_SUBSYSTEM_ID" },
-  {       0, 0x44, 4, "CB_LEGACY_MODE_BASE" },
+  {       0, 0x10, 4, "BASE_ADDRESS_0", 0x3 },
```

Comment:
This breaks alignment of table columns. Please put the type mask before the name.

Reply:
Done, updated.